v3/integration/result.go

```diff
@@ -9,10 +9,10 @@ import (
 )
 
 type resultCount struct {
-	FatalCount  uint8 `json:",omitempty"`
-	ErrCount    uint8 `json:",omitempty"`
-	WarnCount   uint8 `json:",omitempty"`
-	NoticeCount uint8 `json:",omitempty"`
+	FatalCount  uint32 `json:",omitempty"`
```

Comment:
![image](https://user-images.githubusercontent.com/8518514/106221097-ca78d200-6191-11eb-9ef8-544313256c78.png)

I'm gonna pull this branch and give it a quick smoke check myself as well as double check the sanity if the new values.

Reply:
Thank you! Take your time :pray: